pages/data-design/start-using-inspector.mdx

```diff
@@ -34,13 +34,27 @@ Inspector does not process property values or PII data, only event names, proper
 
 TL;DR: No PII data ever flows through Avo servers; only event schemas.
 
-## How to use Inspector
+## Learning about Inspector issues
 
 After <Link passHref href="/workspace/inspector#configuring-sources"><a>setting up your sources</a></Link> to send data to the Inspector with the direct CDP integrations or <Link passHref href="/implementation/avo-inspector-sdk-reference"><a>Avo Inspector SDKs</a></Link> you will see the data appearing in the the Inspector tab.
+There are two overviews available Inspector your incoming data and issues, the <Link passHref href="/workspace/inspector/inspector-issues-view"><a>Issues view</a></Link> and the <Link passHref href="/workspace/inspector/inspector-events-view"><a>Events view</a></Link> – and you can set up <Link passHref href="/workspace/inspector/inspector-slack-alerts"><a>Slack Alerts</a></Link> to be notified about new issues.
```

Comment:
> There are two overviews available Inspector your incoming data and issues [...]

Does this pass as a valid sentance? It reads a bit weird to me.

Reply:
haha you are so picky Hugi

Reply:
Ok done!